src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/EnrichmentManagerSpec.scala

```diff
@@ -324,36 +299,11 @@ class EnrichmentManagerSpec extends Specification with EitherMatchers {
       enriched.value.value.map(_.useragent) must beRight("queryparam-useragent")
     }
 
-    "use Useragent HTTP Header if ua query parameter is not set" >> {
+    "use user agent of HTTP header if 'ua' query string parameter is not set" >> {
```

Comment:
It seems that these tests use an empty enrichment registry. It'd be nice to have a registry with enrichments that use user agent (e.g. YAUAA) to prove these enrichments will indeed pick the expected user agent.